Firebase/Core/Private/FIRLogger.h

```diff
@@ -90,7 +90,7 @@ extern void FIRLogBasic(FIRLoggerLevel level,
 
 #ifdef __cplusplus
 extern "C" {
-#endif // __cplusplus
+#endif  // __cplusplus
```

Comment:
Why the extra spaces, is it from clang-format?

Reply:
Yes, it's clang-format.